includes/defaults.inc.php

```diff
@@ -599,7 +600,7 @@
 
 // Distributed Poller-Settings
 $config['distributed_poller']                            = false;
-$config['distributed_poller_name']                       = file_get_contents('/proc/sys/kernel/hostname');
+$config['distributed_poller_name']                       = trim(file_get_contents('/proc/sys/kernel/hostname'));
```

Comment:
:+1: well spotted, didnt know `file_get_contents()` adds a `\n` to the hostname.